cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden.
+    if xblock.category == 'sequential' and getattr(parent_xblock, "is_entrance_exam", False):
```

Comment:
You don't check here if parent_xblock is already defined, though you do on line 790. What will happen if parent_xblock is None?

I'm a little confused about making the subsection not visible, as I would think that also hides the units. Perhaps the rest of the code will make this clearer.